analysis/report_templates/default.html

```diff
@@ -303,6 +303,12 @@ <h2 class="green-text text-darken-3">experiment data</h2>
             The experiment was conducted using this FuzzBench commit:
             <a href="https://github.com/google/fuzzbench/commits/{{ experiment.git_hash }}">{{ experiment.git_hash }}</a>
             {% endif %}
+
+            {% if description %}
```

Comment:
I dont think anyone will find this at bottom, shouldnt it be at the top of page ? if you still prefer to keep in this last section, it should be the starting part in this section ?

Reply:
>I dont think anyone will find this at bottom, shouldnt it be at the top of page ?

Probably not but I think it should go in this section. Maybe the section should be moved up?

I don't know. I think putting it last makes it easy to tell when the description ends. Maybe if we had some way of description from other text it could be moved up in this section. 
I'm totally fine if someone wants to move this later, but I don't really want to get into heavy frontend work in this PR.